src/shared/HtmlReady.js

```diff
@@ -142,8 +142,8 @@ function link(state, child) {
     if (url) {
         state.links.add(url);
         if (state.mutate) {
-            // If this link is not relative, http, https, or steem -- add https.
-            if (!/^((#)|(\/(?!\/))|(((steem|https?):)?\/\/))/.test(url)) {
+            // If this link is not relative, http, https, steem or esteem -- add https.
+            if (!/^((#)|(\/(?!\/))|(((steem|esteem|https?):)?\/\/))/.test(url)) {
```

Comment:
The comment references `http` but the regex doesn't match `http`. Is that intentional?

Reply:
@JonathanPorta `http` will match `/https?/` :)